src/server/routers/_app.ts

```diff
@@ -242,6 +242,21 @@ export const appRouter = router({
     .mutation(async ({ ctx, input }) => {
       const res = await axios.post(process.env.APPLICATION_URL as string, input)
     }),
+
+  acceptProposalApplication: publicProcedure
+    .input(
+      z.object({
+        proposalId: z.string(),
+        proposalApplicationId: z.string(),
+        applicantEmail: z.string().email(),
+      })
+    )
+    .mutation(async ({ ctx, input }) => {
+      const res = await axios.post(
+        process.env.APPLICATION_ACCEPTANCE_URL as string,
+        input
```

Comment:
extend the input with a process.env.FLOW_SECRET, that is stored in doppler/deployment config, and checked with a control statement in the flow. to ensure nobody can randomly trigger the flow

Reply:
How to do this. Need help. Could not just add it as another input value --> was undefined in the frontend & don't want to use NEXT_PUBLIC